Makefile

```diff
@@ -144,21 +157,22 @@ kind-create: kind-clean
 OLM_IMAGE := quay.io/operator-framework/olm:local
 deploy:
 	$(KIND) load docker-image $(OLM_IMAGE) --name $(KIND_CLUSTER_NAME); \
-	$(HELM) install olm deploy/chart \
+	$(HELM) upgrade --install olm deploy/chart \
 		--set debug=true \
 		--set olm.image.ref=$(OLM_IMAGE) \
 		--set olm.image.pullPolicy=IfNotPresent \
 		--set catalog.image.ref=$(OLM_IMAGE) \
 		--set catalog.image.pullPolicy=IfNotPresent \
+		--set catalog.commandArgs=--configmapServerImage=$(CONFIGMAP_SERVER_IMAGE) \
+		--set catalog.opmImageArgs=--opmImage=$(OPERATOR_REGISTRY_IMAGE) \
 		--set package.image.ref=$(OLM_IMAGE) \
 		--set package.image.pullPolicy=IfNotPresent \
 		$(HELM_INSTALL_OPTS) \
 		--wait;
 
 .PHONY: e2e-build
 e2e-build: BUILD_TAGS="json1 e2e experimental_metrics"
-e2e-build: export GOOS=linux
-e2e-build: export GOARCH=amd64
```

Comment:
Are you sure you don't need the `export`? We added that recently because it was causing problems with test builds on macs

Reply:
I think we can drop the GOARCH as that could screw up M-class cpu mac people
I'll fix GOOS tho =D